packages/server/safetyPatchRes.ts

```diff
@@ -5,7 +5,7 @@ const safetyPatchRes = (res: HttpResponse) => {
     throw new Error('already patched')
   }
   res.onAborted(() => {
-    res.done = true
+    res.aborted = true
```

Comment:
we now have a separate flag for when the client aborts the connection (socket hangup, et al).
this is useful because now if we see a warning in the console we know it wasn't from a client abort.

we use this safetyPatchRes file so we don't have to worry about client aborts. otherwise we'd have to check for an abort after every single async call (see handleUpgrade.ts) 

However, we shouldn't be calling a terminating event after res.done is true. That's a bug in our own business logic